src/main/java/core/basesyntax/IsoscelesTrapezoid.java

```diff
@@ -0,0 +1,24 @@
+package core.basesyntax;
+
+public class IsoscelesTrapezoid extends Figure {
+    private double median;

```

Comment:
better include two parallel sides, the median can be calculated based on them. How we don't have complete info about a figure